<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtorient</class>
 <widget class="QWidget" name="qtorient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtOrient</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Accelerometer</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Y</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="accel_y">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Poll period</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="modeBox"/>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="pollEdit"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Inclination</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="incl_x">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Z</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Orientation</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="incl_y">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="accel_x">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="incl_z">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Mode</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QComboBox" name="orientBox"/>
   </item>
   <item row="7" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
